app/config/collections.php

```diff
@@ -3054,7 +3054,29 @@
                 'required' => false,
                 'default' => null,
                 'filters' => [],
-            ]
+            ],
+            [
+                'array' => false,
+                '$id' => ID::custom('memory'),
+                'type' => Database::VAR_INTEGER,
+                'format' => '',
+                'size' => 512,
+                'signed' => false,
+                'required' => true,
```

Comment:
These attributes should not be required. 

Else we can't to deploy this change without a migration